progs/tests/tuples_fail_invalid_singleton.sy

```diff
@@ -0,0 +1,7 @@
+start :: fn {
+    a := (,)
+    a
+}
+
+//TODO(gu) See #100
+// errors: [ErrorKind::SyntaxError(_, _), ErrorKind::SyntaxError(_, _), ErrorKind::SyntaxError(_, _)]
```

Comment:
#100

Reply:
We should remove parse_branch.

Reply:
Well, yes. But I think it is out of scope here.